npg_bank_account_reconciliation/npg_bank_account_reconciliation.py

```diff
@@ -236,13 +240,14 @@ def refresh_record(self, cr, uid, ids, context=None):
                 line.move_line_id.id
                 for line in obj.credit_move_line_ids + obj.debit_move_line_ids
                 if line.move_line_id
-            ]
+            ]            
 
             domain = [
                 ('id', 'not in', move_line_ids),
                 ('account_id', '=', obj.account_id.id),
                 ('move_id.state', '=', 'posted'),
                 ('cleared_bank_account', '=', False),
+                ('journal_id', 'not in', opening_journal_ids)
```

Comment:
We could probably just add this domain `('journal_id.type', '!=', 'situation')` instead of fetching all journals first.